src/api/services/publishDocuments.spec.js

```diff
@@ -222,6 +227,50 @@ describe('publishDocuments', () => {
 
             expect(transform).toHaveBeenCalledWith(doc);
         });
+
+        it('should add hiddenResource to document if it exists', async () => {
+            const transform = jest.fn().mockImplementation(() => ({
+                uri: 'uid:/09P2JFN2',
+                transformed: 'data',
+            }));
+            const doc = {
+                _id: 'id',
+                uri: 'uri',
+                data: 'value',
+            };
+            const date = new Date();
+            const hiddenResource = [
+                {
+                    uri: 'uid:/09P2JFN2',
+                    reason: 'reason',
+                    removedAt: date,
+                },
+                {
+                    uri: 'uid:/0R4JCK4F',
+                    reason: 'reason',
+                    removedAt: date,
+                },
+            ];
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏You could use two different values for `reason`, which would allow to check that the one you get in the result is indeed the one matching the resource uri.